src/components/blocks/latestItemsBlock/LatestItemsBlock.tsx

```diff
@@ -0,0 +1,44 @@
+import { LinkWithQuery } from "@/components/LinkWithQuery";
+import { TLatestItem } from "@/types";
+import { formatDate } from "@/utils/timedate";
+
+export interface IProps {
+  latestItems: TLatestItem[];
+}
+
+export const LatestItemsBlock = ({ latestItems: latestItems }: IProps) => {
+  const ITEM_LIMIT = 3;
+  const limitedLastItems = latestItems.slice(0, ITEM_LIMIT);
+
+  return (
+    <aside aria-labelledby="latest" className="w-[321px] h-[520px] bg-[#E8EBED] p-4 shadow flex flex-col">
+      <h2 id="latest" className="text-m font-bold text-text-primary mb-4 border-b border-gray-300 pb-4">
+        Latest
+      </h2>
+
+      <div className="flex-1 overflow-auto">
+        <ol className="list-none space-y-4 min-w-0">
+          {limitedLastItems.map((item) => {
+            const [year, day, month] = formatDate(item.date);
+            return (
+              <li key={item.slug} className="border-b border-gray-300 pb-4 min-w-0">
```

Comment:
Request: we are trying to migrate border colours over to the new theme values in the theme.css file, they are listed as `border-[variant]`, please use one of these that fits.